File: 240428/키를 기준으로 정렬/sort-by-height.py
n = int(input())

class person:
    def __init__(self, name, height, weight):
        self.name = name
        self.height = height
        self.weight = weight

people = []

for i in range(n):
    name, height, weight = input().split()
    people.append(person(name, height, weight))

people.sort(key=lambda x : x.height)

for person in people:
    print(f"{person.name} {person.height} {person.weight}")

File: 240630/키를 기준으로 정렬/sort-by-height.js
class Student{
    constructor(name, height, weight){
        this.name = name
        this.height = height
        this.weight = weight
    }
}


const fs = require('fs')
let input = fs.readFileSync(0).toString().trim().split('\n')

let n = Number(input[0])

let students = []

for(let i=1; i<n+1; i++){
    let [n, h, w] = input[i].split(' ')
    students.push(new Student(n, Number(h), Number(w)))
}

students.sort((a,b)=>{
    return a.height - b.height
})

for(let i=0; i<n; i++){
    console.log(students[i].name, students[i].height, students[i].weight)
}